roles/dtc/create/tasks/sub_main_vxlan.yml

```diff
@@ -69,6 +69,7 @@
     - name: Config-Save for Fabric {{ MD_Extended.vxlan.fabric.name }} - Failed
       ansible.builtin.debug:
         msg: "{{ config_save.msg.DATA }}"
+      when: config_save.msg.RETURN_CODE is defined and config_save.msg.RETURN_CODE == 500
```

Comment:
Not sure I understand why we need this `when:` condition.  We should only execute the task in the `rescue:` block IF the task on line 61 fails for some reason.  Is that not the case/